dipy/tracking/tests/test_utils.py

```diff
@@ -661,5 +663,22 @@ def test_min_at():
     npt.assert_array_equal(a, [[[100, 11, 1, 10]]])
 
 
+def test_curvature_angle():
+    angle = [0.0000001, np.pi/3, np.pi/2]
+    step_size = [0.2, 0.5, 1.5]
+    curvature = [2000000., 0.5, 1.060660171]
+
+    for theta, step, curve in zip(angle, step_size, curvature):
+        res_angle = max_angle_from_curvature(curve, step)
+        npt.assert_almost_equal(res_angle, theta)
+
+        res_curvature = min_curvature_from_angle(theta, step)
+        npt.assert_almost_equal(res_curvature, curve)
+
+    # special case
+    npt.assert_equal(min_curvature_from_angle(0, 1),
+                     min_curvature_from_angle(np.pi/2, 1))
+
+
 if __name__ == "__main__":
-    npt.run_module_suite()
```

Comment:
This should be reverted, correct?

Reply:
no, we should remove them all. it is using nosetest which is deprecated. we do not use it anymore and I don't know if NumPy will update it

Reply:
Why does the main block only run this one function then? Should it run the other tests defined here as well? Or should we delete all the main blocks in test files?

Reply:
Personally, I would recommend removing all the main blocks in test files. But, this is something to discuss and check. I do not know if doing the code below is still working:
```python
import dipy
dipy.test()
```

I will remove this one here, good catch and thank you for the feedback

Reply:
here a link concerning `tester`: https://numpy.org/doc/stable/reference/generated/numpy.testing.Tester.html#numpy.testing.Tester

Reply:
and `run_module_suite`: https://numpy.org/doc/stable/reference/generated/numpy.testing.run_module_suite.html#numpy.testing.run_module_suite

We should create an issue to remove it all since it depends on `nosetests`

Reply:
👍 No rush! My approach, and I imagine others are doing this as well was to use pytest to run the tests, so there is a viable alternative. This would just be good to clean up.